nni/runtime/platform/local.py

```diff
@@ -21,9 +20,6 @@
     os.makedirs(_outputdir)
 
 _nni_platform = trial_env_vars.NNI_PLATFORM
-if _nni_platform == 'local':
```

Comment:
remove trial.log in local mode?

Reply:
The logging system is refactored. Its initialized when `nni` (top-level package) get imported.